### ISSUE TITLE
Cannot open details page in subfolders for newly added bookmarks

### ISSUE THREAD

**Issue by e-alfred** [nextcloud/bookmarks#814]
**Describe the bug**
If I enter a subfolder in the bookmarks app and try to open the details page of a newly added bookmark from the "..." menu, it wont do anything, except showing an error in the browser console.

**To Reproduce**
1. Go into a subfolder in the bookmarks app.
2. Click on "details" on a newly added bookmark in a subfolder.

**Expected behavior**
The details page for a newly added bookmark should be opened.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Debian Buster
 - Browser Firefox
 - Version 69.0.1

**Server (please complete the following information):**
 - OS: Ubuntu 18.04
 - HTTP server: Apache 2.4
 - Database: Mysql 5.7
 - PHP version: 7.2
 - Nextcloud version: 16.0.4
 - Bookmarks app version: 2.1.1
 - Nextcloud external user backend: none

**Additional context**
The bookmarks were synced using Floccus. Old bookmarks in the folder show no problems.

**Browser log**
```
Error: "No language found."
    u https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:713
    e https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:713
    e https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:713
    addedDate https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:947
    get https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    evaluate https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    yn https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    ze https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:947
    _render https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    i https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    get https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    run https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    dn https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    ee https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    Xt https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
bookmarks.main.js:7:11622
    Yt https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    Ut https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    Ft https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    _render https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    i https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    get https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    run https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    dn https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    ee https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7
    Xt https://myserver.domain/cloud/apps/bookmarks/js/bookmarks.main.js?v=08de0921-1:7

```

**Comment by j-ed** [comment]
I can confirm this behavior but it happens independently from any sub-folder. The details cannot be opened nor edited.

**Comment by marcelklehr** [comment]
EIther I've inadvertantly fixed this already, or I can't reproduce this. For me the new bookmark opens automatically in the detail view.

**Comment by j-ed** [comment]
@marcelklehr At least on my server I cannot reproduce the problem with the latest app version installed.

**Comment by marcelklehr** [comment]
@j-ed That's great news :)

**Comment by j-ed** [comment]
Unfortunately I have to report, that the problem appears again with the latest app version 2.2.0.
If I, e.g. add the following url, I'm not able to open the "`Details`" dialog afterwards but "Renaming",
"Moving" and "Deleting" are working without a problem:

https://www.kochbar.de/rezept/455462/Orangentorte/drucken/personen/12.html

![grafik](https://user-images.githubusercontent.com/2572764/66706001-0c2c0200-ed2e-11e9-8d39-b75dd7f777cd.png)

**Comment by paulcalabro** [comment]
I'm not able to reproduce this issue with NC 17 and v2.2.0 of the app. My "Details" tab opens fine:

![Screen Shot 2019-10-12 at 11 45 21 AM](https://user-images.githubusercontent.com/585059/66706329-ef6bdc00-ece5-11e9-97ed-8515396451c6.png)

**Comment by marcelklehr** [comment]
This is likely related to the language of your interface. I recently fixed a bug that presented similarly.

**Comment by paulcalabro** [comment]
Ahhh, gotcha! Yeah, I'm using English (US).

**Comment by alastair-dm** [comment]
Can we reopen this one please? I'm still seeing it in NC17 and 2.2.0, at least when my Language is set to "English (British English)". If I change the Language setting for my user to "English (US)" the "Details" tab opens and I can edit. Change back to "English (British English)" and "Details" no longer works for the same set of "new" tags as before.

**Comment by marcelklehr** [comment]
I'm sorry, I wasn't clear on the path ahead: I just released v2.3.0, which fixes this.

**Comment by alastair-dm** [comment]
Thanks. I'll try it when the update appears - currently update:check says "Everything up to date" so I guess I'll have to wait a bit.

**Comment by alastair-dm** [comment]
After updating to v2.3.0 "Details" works for me with the language set to "English (British English)", with the added bonus of the whole interface responding much faster. Thanks again.

**Comment by e-alfred** [comment]
My language setting is German as well.

**Comment by j-ed** [comment]
I've just installed the new version and can confirm that it is possible to open the details dialog again, **except** that all previously defined keywords are not known anymore.
I can see previously assigned keywords of older bookmarks, but as soon as I select the input field they're not shown anymore and "List is empty" is displayed. @alastair-dm , @e-alfred can you confirm this behavior?

**Comment by alastair-dm** [comment]
I can confirm that initial behaviour having force-reloaded the bookmarks page. The tags list on both the sidebar and the details tab is unpopulated until I add a tag to a bookmark. At that point both the sidebar and the details tab gain the full list of existing tags. I had assumed this was just slow background loading as I quickly edited a bookmark I hadn't been able to tag before, but having revisited it the loading of the tag list seems not to happen until I add a tag to a bookmark.

**Comment by j-ed** [comment]
@alastair-dm Yes, you're right, the list of tags is only displayed if one has manually been added. 
I will create a new issue to address this behavior.

**Comment by e-alfred** [comment]
Yes, I can confirm that no tags are shown anymore.

**Comment by alastair-dm** [comment]
> Yes, I can confirm that no tags are shown anymore.

That's tracked in #840 and has already been fixed in v2.3.1
